source/_docs/create-custom-upstream.md

```diff
@@ -259,11 +259,20 @@ To avoid incompatibilities, you must track Pantheon's corresponding upstream rep
 <div role="tabpanel" class="tab-pane" id="gitlab-https" markdown="1">
 Custom Upstreams from GitLab repositories must be created for you by Pantheon Support.
 
-1. Prepare a GitLab user with at least **Reporter** level permissions for your upstream repository.
+1. Prepare a new GitLab user specifically for Pantheon's use, with at least **Reporter** level permissions for your upstream repository:
```

Comment:
It might be clearer to say only "Create a new user for Pantheon's use" and leave the reporter part (and the image) for below when the user is added to the repo -- users don't have roles until they're members of something.